src/overlay025/poketch_system.c

```diff
@@ -128,68 +128,68 @@ void PoketchSystem_Create(FieldSystem *fieldSystem, PoketchSystem **poketchSys,
     if (new_system != NULL) {
         *poketchSys = new_system;
 
-        new_system->fieldSystem = fieldSystem;
+        new_system->fieldSys = fieldSys;
         new_system->saveData = saveData;
         new_system->poketchData = SaveData_PoketchData(saveData);
         new_system->bgl = bgl;
-        new_system->unk_50 = oamManager;
+        new_system->oamManager = oamManager;
 
         if (PoketchSystem_InitInternal(new_system)) {
             sub_0201E3D8();
             sub_0201E450(4);
 
             new_system->poketchSysPtr = poketchSys;
-            new_system->unk_38 = SysTask_Start(ov25_02253E94, new_system, 4);
+            new_system->unk_38 = SysTask_Start(PoketchSystem_PostRender, new_system, 4);
 
             SysTask_Start(PoketchSystem_MainTask, new_system, 0);
         }
     }
 }
 
-void ov25_02253D5C (PoketchSystem *poketchSys)
+void PoketchSystem_StartShutdown (PoketchSystem *poketchSys)
 {
-    if ((poketchSys->unk_00 != 3) && (poketchSys->unk_00 != 4)) {
-        ov25_02253E9C(poketchSys, 3);
+    if ((poketchSys->systemState != POKETCHSYSTEM_SHUTDOWN) && (poketchSys->systemState != POKETCHSYSTEM_UNLOAD)) {
+        PoketchSystem_SetState(poketchSys, POKETCHSYSTEM_SHUTDOWN);
     }
 }
 
-BOOL ov25_02253D70 (PoketchSystem *poketchSys)
+BOOL PoketchSystem_IsSystemShutdown(PoketchSystem *poketchSys)
 {
     return poketchSys == NULL;
 }
 
-void ov25_02253D7C (PoketchSystem *poketchSys, int param1, u32 param2)
+void PoketchSystem_SendFieldEvent(PoketchSystem *poketchSys, enum PoketchFieldEventID eventID, u32)
```

Comment:
What does a "field event" refer to here? Sounds different from that in `res/field/events`

Reply:
Renamed it to `PoketchSystem_SendEvent`.

Reply:
Rename the enum to `PoketchEventID`.